tests/rptest/tests/pandaproxy_test.py

```diff
@@ -101,8 +101,18 @@ def _get_topics(self, headers=None):
             headers = self._get_topics_headers()
         return requests.get(f"{self._base_uri()}/topics", headers=headers)
 
-    def _produce_topic(self, topic, data):
-        return requests.post(f"{self._base_uri()}/topics/{topic}", data)
+    def _produce_topics_headers(self):
+        return {
```

Comment:
sounds like this should be a global object ? or you intend go mod this.

Reply:
Done